src/core/examples.js

```diff
@@ -62,14 +62,12 @@ function makeTitle(elem, num, report) {
   if (report.title) elem.removeAttribute("title");
   const number = num > 0 ? ` ${num}` : "";
   const title = report.title
-    ? html` <span class="example-title">: ${report.title}</span> `
+    ? html`<span class="example-title">: ${report.title}</span>`
     : "";
-  return html`
-    <div class="marker">
-      <a class="self-link">${l10n.example}<bdi>${number}</bdi></a
-      >${title}
-    </div>
-  `;
+  return html`<div class="marker">
```

Comment:
On a second thought, maybe extra indentation was better. Looks a bit sticky. WDYT?

Reply:
I personally like this as:

* Consistency. We definitely don't want spaces in cases like `` html` <img src="" /> ` ``, so it's consistent to apply the same rule.
   * 2.0 will collapse newlines into a space if the element is not too long: `\n<div>...</div>\n` can potentially be collapsed into ` <div>...</div> `, where we certainly don't want the spaces. This can prevent such fluctuation.
* Does not depend on trimming behavior from a specific library. AFAIK not all libraries do this, e.g. `lit-html` from Google.

Reply:
There was a similar opinion when I open the prettier PR, though. The conclusion was: let's make this right first and decide further styling later.

Reply:
One reason I was opposing this is that there are too many (avoidable) changes caused by this, which makes it slightly difficult for git blame. Fine with what you decide though.

> Does not depend on trimming behavior from a specific library. 

👍 💯

Reply:
>which makes it slightly difficult for git blame

True that. But we shouldn't use prettier in that case 😂, like, the big changes it forced to ReSpec when it first introduced HTML template formatting...

Reply:
That's what we get for opinionated code formatters 😄